### PR TITLE
feat(certManager): update to 1.3.2

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Updating cert manager to include following fix https://github.com/newrelic/k8s-webhook-cert-manager/releases/tag/v1.3.2

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       E2E_MINIKUBE_DRIVER: docker
     strategy:
       matrix:
-        k8s_version: [ "v1.15.12", "v1.16.15", "v1.17.14", "v1.18.12" ]
+        k8s_version: [ "v1.15.12", "v1.16.15", "v1.17.14", "v1.18.12", "v1.19.4" ]
```
Comment (roobre) [inline .github/workflows/build.yml:35]: Finally! 🥲

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.3.2
+
+- Update k8s-webhook-cert-manager to 1.3.2
```
Comment (roobre) [inline CHANGELOG.md:9]: I would note explicitly that we now support 1.19

Comment (paologallinaharbur) [reply CHANGELOG.md:9]: updated @roobre

**Comment by roobre** [comment]
I think you might have missed these:

https://github.com/newrelic/k8s-metadata-injection/blob/7e68256e41598bdcca4e8b98a87fae33bc8711d1/Makefile#L5

https://github.com/newrelic/k8s-metadata-injection/blob/66436f7815549d1c44918e796c94d3c9d2062282/deploy/newrelic-metadata-injection.yaml#L22

**Comment by paologallinaharbur** [comment]
You are right, in this repo the version change is not automatic

**Review by roobre — APPROVED** [review]
lgtm

**Pull request merged**